### PR TITLE
sl-remove-overlay-on-resize

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
Addresses https://github.com/dart-lang/dartdoc/issues/742

**Comment by sethladd** [comment]
cc @devoncarew

File: lib/resources/styles.css [hunk 1]
```diff
@@ -562,6 +562,17 @@ button {
     margin-right: 24px;
   }
 
+  #overlay-under-drawer.active {
+    opacity: 0.7;
+    width: 100%;
```
Comment (devoncarew) [inline lib/resources/styles.css]: can also do `right: 0; bottom; 0` instead of width and height

File: lib/resources/styles.css [hunk 2]
```diff
@@ -562,6 +562,17 @@ button {
     margin-right: 24px;
   }
 
+  #overlay-under-drawer.active {
```
Comment (devoncarew) [inline lib/resources/styles.css]: What properties does `#overlay-under-drawer` have if not active? It looks like a 0 width height div. Perhaps have it be display: none?

**Comment by devoncarew** [comment]
lgtm

File: lib/resources/styles.css (hunk 1 above)
Comment (sethladd) [inline lib/resources/styles.css]: Happy to change it if that's better. Are you saying to change it? :)

File: lib/resources/styles.css (hunk 2 above)
Comment (sethladd) [inline lib/resources/styles.css]: Good call. I'll make sure it's display: none.

File: lib/resources/styles.css (hunk 1 above)
Comment (devoncarew) [inline lib/resources/styles.css]: I would. I don't really trust percentages w/ the dom. I think the fixed positioning will be more reliable.

**Comment by sethladd** [comment]
lgtm

**Pull request closed without merging**